### PR TITLE
[SC-307] Create account for BMGF

### PR DESCRIPTION
**Pull request by zaro0508**
Create an AWS account to run a Service Catalog for the Bill and
Melinda gates foundation.

### CODE REVIEW

File: org-formation/organization.yaml [hunk 1]
```diff
@@ -230,6 +231,15 @@ Organization:
       Tags:
         <<: !Include ./_default_org_tags.yaml
 
+  BMGFAccount:
+    Type: OC::ORG::Account
+    Properties:
+      AccountName: org-sagebase-bmgf
+      RootEmail: aws.bmgf@sagebase.org
```
Comment (zaro0508) [inline org-formation/organization.yaml:238]: group created, https://groups.google.com/a/sagebase.org/g/aws.bmgf

**Comment by zaro0508** [comment]
@brian-bot would you mind giving this a review.

**Comment by zaro0508** [comment]
@brian-bot this is for issue SC-307 basically to setup an account to host a service catalog for BMGF.

File: org-formation/organization.yaml [hunk 2]
```diff
@@ -230,6 +231,15 @@ Organization:
       Tags:
         <<: !Include ./_default_org_tags.yaml
 
+  BMGFAccount:
+    Type: OC::ORG::Account
+    Properties:
+      AccountName: org-sagebase-bmgf
```
Comment (zaro0508) [inline org-formation/organization.yaml:237]: @brian-bot should this account be called `bmgf` or should it be more specific, like some sort of project name?

**Review by brucehoff — APPROVED** [review]

File: org-formation/organization.yaml (hunk 2 above)
Comment (zaro0508) [reply org-formation/organization.yaml]: Done

**Pull request merged**